deploy/operator/base/rbac.yaml

```diff
@@ -5,6 +5,10 @@ metadata:
   labels:
     rbac.authorization.k8s.io/aggregate-to-edit: "true"
 rules:
+  - apiGroups: ["security.openshift.io"]
```

Comment:
I think this is a remnant of my attempt to fix the artifact cache perm issue

Reply:
unable to test locally (still in kcp debug) but I removed this remnant just to eliminate it as a variable

let's see what CI says